object_filter_node:
  ros__parameters:
    input_topic: "/objects_on_route"
    output_topic: "/objects_filtered"
    excluded_labels: [1, 2, 6]  # 番号要確認
